### ISSUE TITLE
How to install

### ISSUE THREAD

**Issue by liliwise** [nimrodbusany/Radiant4GoogleReps#1]
In your PDF, I don't know how to install this tools. Is there a detailed document? 
Thanks a lot!

**Comment by nimrodbusany** [comment]
It seems that we haven't included a list of packages. I remember using
Shiny R and shiny.
Did you try to follow the installation instructions of the original
project, https://github.com/vnijs/radiant/blob/master/README.md ?
Our project was just an extension with a few fancy visualizations.

Bests,
Nimrod

**Comment by liliwise** [comment]
Yes, I'm trying to install the radiant, But I encountered some questions, so I submitted the question on their issue. Whatever, thank you very much for your reply.

**Comment by nimrodbusany** [comment]
Sure, I hope you manage to fix it.

Bests,
Nimrod
